### PR TITLE
[script] make shallow clone of submodules

### PR DESCRIPTION
**Pull request by wgtdkp**

### CODE REVIEW

File: script/bootstrap.sh [hunk 1]
```diff
@@ -128,5 +128,5 @@ readonly CUR_DIR="$(dirname "$(realpath -s $0)")"
 readonly PROJECT_DIR=${CUR_DIR}/../
 
 cd ${PROJECT_DIR}
-git submodule update --init --recursive
+git submodule update --init --recursive --depth=1
```
Comment (simonlingoogle) [inline script/bootstrap.sh]: any reason for doing it ?

Comment (wgtdkp) [reply script/bootstrap.sh]: This can speed up the submodule cloning process.

Comment (simonlingoogle) [reply script/bootstrap.sh]: Thanks. New skill got. :+1:

**Comment by jwhui** [comment]
# [Codecov](https://codecov.io/gh/openthread/ot-commissioner/pull/24?src=pr&el=h1) Report
> Merging [#24](https://codecov.io/gh/openthread/ot-commissioner/pull/24?src=pr&el=desc) into [master](https://codecov.io/gh/openthread/ot-commissioner/commit/5326434f017dfdafb959b9c2f78487839d63409d&el=desc) will **not change** coverage by `%`.
> The diff coverage is `n/a`.

```diff
@@           Coverage Diff           @@
##           master      #24   +/-   ##
=======================================
  Coverage   59.94%   59.94%           
=======================================
  Files          54       54           
  Lines        5400     5400           
=======================================
  Hits         3237     3237           
  Misses       2163     2163           
```

**Pull request merged**